source/cli.rst

```diff
@@ -2302,6 +2302,10 @@ as a child.
 if you think that the child CA may be temporarily disabled, then you may wish to
 "suspend" them instead.
 
+.. Important:: If the child CA was created in krill using the `krillc children add --ca testbed --child newca` 
```

Comment:
Actually.. I would say:

Note that this action just removes the child from the given CA parent. The child CA as such is not removed by this action. Obviously, if the child is a remote CA then your krill instance cannot possible remove that CA. But, even if the child is also hosted in your own Krill instance, then you will need to explicitly delete it using `krillc delete --ca <ca_name>` if that is what you intended to do.